### PR TITLE
New package: ktouch-18.08.3

### PR DESCRIPTION
**Pull request by 1is7ac3**

### CODE REVIEW

File: srcpkgs/ktouch/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'ktouch'
+pkgname=ktouch
+version=18.08.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules pkg-config"
+makedepends="libxkbfile-devel kdeclarative-devel kcmutils-devel qt5-xmlpatterns-devel qt5-tools-devel kio-devel qt5-plugin-mysql qt5-plugin-tds qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite"
```
Comment (maxice8) [inline srcpkgs/ktouch/template]: cut at 80 columns

File: srcpkgs/ktouch/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Is a program to learn and practice touch typing"
```
Comment (maxice8) [inline srcpkgs/ktouch/template]: Learn and practive touch typing

File: srcpkgs/ktouch/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+maintainer="1is7ac3 <isaac.qa13@gmail.com>"
+license="GPL-i2.0-or-later"
+homepage="https://kde.org/applications/education/ktouch"
+distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=0bf18f7b4f5644aa1d1f2248577d1aae2a96f091abefed9f8334124b60e73ce7
```
Comment (maxice8) [inline srcpkgs/ktouch/template]: add blank line between the variables and the `if` control structure

File: srcpkgs/ktouch/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+version=18.08.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules pkg-config"
+makedepends="libxkbfile-devel kdeclarative-devel kcmutils-devel 
+qt5-xmlpatterns-devel qt5-tools-devel kio-devel qt5-plugin-mysql qt5-plugin-tds 
+qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite"
+short_desc="Learn and practive touch typing"
+maintainer="1is7ac3 <isaac.qa13@gmail.com>"
+license="GPL-i2.0-or-later"
```
Comment (Hoshpak) [inline srcpkgs/ktouch/template]: typo here

File: srcpkgs/ktouch/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'ktouch'
+pkgname=ktouch
+version=18.08.3
```
Comment (Hoshpak) [inline srcpkgs/ktouch/template]: 18.12.0 has already been release, please update the template.

**Comment by 1is7ac3** [comment]
the changes have been applied, thanks

File: srcpkgs/ktouch/template (hunk 5 above)
Comment (maxice8) [reply srcpkgs/ktouch/template]: @1is7ac3 ^

**Comment by maxice8** [comment]
Please update to new version 18.12.0

**Comment by 1is7ac3** [comment]
version 18.12.0 failed
/bin/sh: /usr/bin/qmlcachegen: No such file or directory
not supplied by qt5-declarative

**Comment by Hoshpak** [comment]
This is a bug in qt5 that has been fixed in 5.12.0: https://bugreports.qt.io/browse/QTBUG-68724

The options here are:
1) Merge the old version and update once qt5 gets updated
2) Temporarily add a workaround in the template (a symlink should do it) and use the current version
3) Patch the qt5 version in the repository to set the path correctly and use the current version

**Comment by 1is7ac3** [comment]
version  18.12.1 with a symlink

File: srcpkgs/ktouch/template [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'ktouch'
+pkgname=ktouch
+version=18.12.1
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules pkg-config"
+makedepends="libxkbfile-devel kdeclarative-devel kcmutils-devel
+qt5-xmlpatterns-devel qt5-tools-devel kio-devel qt5-plugin-mysql qt5-plugin-tds
```
Comment (maxice8) [inline srcpkgs/ktouch/template]: needs one whitespace indent

File: srcpkgs/ktouch/template [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+qt5-quickcontrols2-devel"
+short_desc="Learn and practive touch typing"
+maintainer="1is7ac3 <isaac.qa13@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://kde.org/applications/education/ktouch"
+distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=4529efa94dc0574790f492e9cc54ed898596c424f6444e025f81d0427a87746c
+
+pre_configure() {
+ _dirQml=$(which qmlcachegen-qt5)
```
Comment (maxice8) [inline srcpkgs/ktouch/template]: please don't modify the state of the system, instead edit the generated makefile to remove $XBPS_CROSS_BASE

**Pull request closed without merging**